lib/opencensus-proto/build.rs

```diff
@@ -0,0 +1,16 @@
+extern crate tower_grpc_build;
+
+fn main() {
+    let iface_files = &["opencensus/proto/agent/trace/v1/trace_service.proto"];
```

Comment:
Is it really necessary to vendor so much for this single dependency?  According to their bazel, it seems to only have a few dependencies.  I'd prefer that we vendor only what's needed unless there's a good argument to do otherwise.

Reply:
suggestions illustrated at https://github.com/linkerd/linkerd2-proxy/pull/345